core/trino-main/src/main/java/io/trino/metadata/Split.java

```diff
@@ -90,4 +93,11 @@ public String toString()
                 .add("lifespan", lifespan)
                 .toString();
     }
+
+    public long getRetainedSizeInBytes()
+    {
+        return INSTANCE_SIZE
```

Comment:
why not accounting `lifespan`? Because we plan to remove it?

Reply:
I don't know why, but somehow I thought that's an enum. Let me add accounting for now.